weareinreach/inreach-catalog
# Organization Owner Edit Page Pt. 1

## Description



- Asana ticket: https://app.asana.com/0/1132189118126148/1181017314910216

UI Skeleton to allow Organization Owner to edit Organization Description

<img width="867" alt="Screen Shot 2020-12-28 at 5 28 28 PM" src="https://user-images.githubusercontent.com/18750139/103262327-83f94300-4959-11eb-87b5-8fe6abbc4359.png">
<img width="885" alt="Screen Shot 2020-12-28 at 5 28 35 PM" src="https://user-images.githubusercontent.com/18750139/103262326-82c81600-4959-11eb-8ad3-2cb6ada1335e.png">

## How to Test



1. Log in as organization owner
2. Find organization in search results
3. Click the Edit Icon (red pencil)
4. Click "Yes, I'll make the edits"
5. Verify text says "Back to view mode" at the top
5.a. Verify link will exit Edit Mode
6. Verify "Edit" with Red Pencil Icon appears next to the Organization Name
6.a. Verify clicking the "Edit" button renders a text form
6.b. Verify form is editable
6.c. Verify changes made in form do not change page's Organization data on "cancel"
7. Save button will not change the data
